web-app/README.adoc

```diff
@@ -90,3 +90,6 @@ Code for the app. Refer to https://angular.io/guide/styleguide#application-struc
 === e2e
 Location for all e2e tests.
 
+=== Errors
```

Comment:
I'd say this isn't really a troubleshooting step, since this is something that needs to happen for the program to work. let's add it under it's own header `Installing Dependencies` at the beginning.

Reply:
At the beginning as in at the beginning of the README?